pom.xml

```diff
@@ -39,6 +39,11 @@
             <artifactId>org.hl7.fhir.r4</artifactId>
             <version>5.4.10</version>
         </dependency>
+        <dependency>
+            <groupId>ca.uhn.hapi.fhir</groupId>
+            <artifactId>hapi-fhir-client</artifactId>
+            <version>7.4.3</version>
+        </dependency>
```

Comment:
@lincmba Is there a reason we are upgrading to this version and using the version we have **[here](https://github.com/opensrp/hapi-fhir-jpaserver-starter)**

Reply:
Not upgrading. Just adding the dependency. What is the recommended version?

Reply:
We use HAPI `6.1.3` [here](https://github.com/opensrp/hapi-fhir-jpaserver-starter/blob/ccbbc65915dec2f6c5127e07c19650cac4bb7276/pom.xml#L17). I am not sure it works with what you want. 

This is me just nitpicking trying to get all the versions of HAPI to be same

Reply:
We need some code in the specific version. thanks @lincmba

Reply:
@dubdabasoduba I need to use this version because there is a package `GenericClient` that's in version `7.4.3` and not `6.1.3`. This is also the version gateway-extensions uses.